package.json

```diff
@@ -27,9 +27,10 @@
   "dependencies": {
     "oargv": "^2.0.0",
     "one-by-one": "^2.0.0",
+    "q": "^1.4.1",
     "sliced": "^1.0.1",
     "tmp": "^0.0.26",
     "ul": "^5.0.0",
     "which": "^1.1.1"
   }
-}
```

Comment:
And of course, don't forget to [add yourself as contributor](https://github.com/IonicaBizau/node-cobol/blob/master/CONTRIBUTING.md#fixing-issues).